cumulus/parachains/integration-tests/emulated/tests/assets/asset-hub-westend/src/tests/hybrid_transfers.rs

```diff
@@ -275,6 +275,155 @@ fn transfer_foreign_assets_from_asset_hub_to_para() {
 	assert_eq!(receiver_rocs_after, receiver_rocs_before + foreign_amount_to_send);
 }
 
+// =================================================================================
+// ======= Reserve Transfers - Sufficient Foreign Asset - AssetHub<>Parachain ======
+// =================================================================================
+/// Reserve Transfers of a sufficient foreign asset from System Parachain to Parachain should
+/// work without using any other token.
```

Comment:
This test transfers WND (native token) from AH to Parachain.

In practice the ecosystem (currently) has the native token registered as a "trust-backed" asset with the "trust" coming from the reserve Relay using either pallet-assets or pallet-tokens. I haven't seen anyone implement it using the new "foreign assets" model.

My point is that this particular emulated test is not practically relevant to the community - and I would change the tests to cover the following real-world usecases:

## Teleport (both ways) Parachain-native (PEN) between Parachain and AH (pay with PEN)

Teleport an asset - registered on AssetHub as foreignAsset - between AH and Parachains (do it both ways):
- use the parachain's native asset (let's say PEN)
- is also registered as foreign on AH
- there is a PEN/WND pool set up on AH

This ^ would be a very realistic use-case of what we want in practice: all parachains register their native tokens as foreign on AH, and using asset-conversion can do cross-chain transfers using their native token alone.

## Reserve-transfer (both ways) ecosystem-native (WND) between Parachain and AH (pay with WND)

This is already done in existing tests, but it is achieved by allowing buying weight with WND directly on Penpal (`TakeFirstAssetTrader`) - you could remove that and set up WND/PEN pool instead on Penpal (so it is more representative of real world). This would showcase how parachains could easily support fees in WND/DOT/KSM while still practically using their native token for buying weight (execution fees).

(basically what you did with `reserve_transfer_pool_assets_from_system_para_to_para()` test, but instead of adding new test using some random theoretical asset for the pool, create WND/PEN pool and make existing tests work without `TakeFirstAssetTrader`)

## Reserve-transfer (both ways) AH trust-backed USDT between two parachains going through AH (pay with USDT)

Again, real-world usecase - USDT used for fees using USDT/??? pools available on each hop.

You already covered this in `reserve_transfer_pool_assets_from_para_to_para_through_asset_hub()` 👍